src/views/manage/components/DatePickerInput.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect, useRef } from 'react';
+import DatePicker from 'react-datepicker';
+import styled from '@datapunt/asc-core';
+import 'react-datepicker/dist/react-datepicker.css';
+
+const DatePickerStyle = styled.div`
+  font-size: 1rem;
+  border: solid 1px #767676;
```

Comment:
No `themeColor` function here?

Reply:
style was not used. I removed it for now.